centreon-open-tickets/tests/e2e/plugins.ts

```
@@ -35,36 +35,27 @@ export default (
   });
 
   on('after:run', (results) => {
-    const testRetries: { [key: string]: boolean } = {};
+    const testRetries: { [key: string]: Number } = {};
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)